source/opt/copy_prop_arrays.h

```diff
@@ -170,11 +174,20 @@ class CopyPropagateArrays : public MemPass {
 
   // Returns the memory object that at some point was equivalent to the result
   // of |construct_inst|.  If a memory object cannot be identified, the return
-  // value is |nullptr|.  The opcode of |extract_inst| must be
+  // value is |nullptr|.  The opcode of |constuct_inst| must be
   // |OpCompositeConstruct|.
   std::unique_ptr<MemoryObject> BuildMemoryObjectFromCompositeConstruct(
       ir::Instruction* conststruct_inst);
 
+  // Returns the memroy object that at some pointer was equivalent to the result
+  // of |insert_inst|.  If a memeory object cannot be identified, the return
+  // value is |nullptr\.  The opcode of |insert_inst| must be
+  // |OpCompositeInsert|.  This function looks for a series of
+  // |OpCompositeInsert| instructions That insert the elements one at a time in
+  // order from beginning to end.
```

Comment:
Check the spelling of 'memory' above.

Reply:
Fix up the comment.  Not bothering with retesting for that.